types/query/filtered_pagination_test.go

```diff
@@ -27,10 +28,11 @@ func (s *paginationTestSuite) TestFilteredPaginations() {
 		balances = append(balances, sdk.NewInt64Coin(denom, 250))
 	}
 
+	balances = balances.Sort()
```

Comment:
why do we need to sort?

Reply:
the send will fail if denoms are not sorted, before balance was set using SetBalance which bypassed this.